apps/iiif/manifests/migrations/0054_manifest_logo_url.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.17 on 2023-02-16 20:43
+
+from django.db import migrations, models
+from django.conf import settings
+
+def populate_logo_urls(apps, schema_editor):
+
+    Manifest = apps.get_model("manifests", "Manifest")
+    manifests = Manifest.objects.all()
+
+    # populate logo_url model field with logo URL
+    for man in manifests:
+        man.logo_url = f'{settings.HOSTNAME}/media/{man.logo}'
+        man.save()
```

Comment:
Sorry, forgot to mention this when we discussed! There is a more efficient way to bulk update records in Django; each `save()` is a DB call, `bulk_update` will batch them instead.

```suggestion
    for man in manifests:
        man.logo_url = f'{settings.HOSTNAME}/media/{man.logo}'
    Manifest.objects.bulk_update(manifests, ['logo_url'])
```

Reply:
Implemented as suggested 👍